app/javascript/src/components/category.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div
+    class="category"
+  >
+    <div
+      class="category__main-product"
+    >
+      <div class="product-card">
+        <product-image
+          :product="selectedProduct"
+          @click.native="clickAction"
+        />
+        <div class="choose-option-text">
+          Elige tu opción
+        </div>
+        <div
+          class="category__product-selector"
+        >
+          <div
+            v-for="(product, index) in category.products"
+            :key="index"
+            class="category__product"
+            :class="{'category__product--selected': index === selectedProductIndex }"
+            @click="changeProduct(index)"
+          >
+            <product-image :product="product"/>
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected a space before '/>', but not found  vue/html-closing-bracket-spacing